File: Famoser.MassPass.Data/Entities/Communications/Response/DownloadContentEntityResponse.cs
using System;
using Famoser.MassPass.Data.Entities.Communications.Response.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Response
{
    public class DownloadContentEntityResponse : ApiResponse
    {
        public Uri DownloadUri { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Response/DownloadContentEntityResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:48
 */

namespace Famoser\MassPass\Models\Response;


use Famoser\MassPass\Models\Response\Base\ApiResponse;

class DownloadContentEntityResponse extends ApiResponse
{
    public $DownloadUri;
}